### ISSUE TITLE
Next/Prev Search Results don't act on current Find-result window

### ISSUE THREAD

**Issue by sasumner** [notepad-plus-plus/notepad-plus-plus#9080]
### Description of the Issue
The menu commands _Next Search Result_ and _Previous Search Result_ act only on the original _Find result_ window, not the active _Find result_window.

### Steps to Reproduce the Issue
1. Create and save a file with the following text:
```
Jack and Jill
Went up the hill
To fetch a pail of water
Jack fell down
And broke his crown,
And Jill came tumbling after.

Up Jack got
And home did trot,
As fast as he could caper;
And went to bed
And plastered his head
With vinegar and brown paper.

When Jill came in
How she did grin
To see Jack's paper plaster;
Mother vexed
Did whip her next
For causing Jack's disaster.
```
2. Do a simple _Find All in Current Document_ search for the text `Jack`
3. Observe:
![image](https://user-images.githubusercontent.com/30118311/97756613-88c48680-1ad1-11eb-8c5d-83f405a17261.png)
4. Right-click in the client area of the _Find result_ window and choose _Find in these found results..._
5. Untick all checkboxes and search for `Jill`
6. Observe:
![image](https://user-images.githubusercontent.com/30118311/97756724-c0cbc980-1ad1-11eb-9dcf-7dbed700132d.png)
7. Click in the main editor window
8. Use the menu commands _Next Search Result_ and _Previous Search Result_ (or F4 / Shift+F4, respectively), several times, randomly.

### Actual Behavior
Commands in step 8 moved between results of the first search, finding `Jack`

### Expected Behavior
Commands in step 8 should move between results of the second search for `Jill`, since it is the _Find result_ window that is active and currently showing to the user.

### Debug Information
Notepad++ v7.9.1   (64-bit)
Build time : Oct 27 2020 - 05:04:37
Path : C:\\..........\npp.7.9.1.portable.x64\notepad++.exe
Admin mode : OFF
Local Conf mode : ON
OS Name : Windows 10 Enterprise (64-bit) 
OS Version : 1809
OS Build : 17763.1457
Current ANSI codepage : 1252
Plugins : mimeTools.dll NppConverter.dll NppExport.dll

**Comment by clachen** [comment]
Here is a potential fix to the issue, which as far as I could discern did not cause any other bugs. 
It was done by reassigning already existent information to the _pFinder pointer for search next result to access. 
#9191

**Comment by donho** [comment]
It's not really a bug to me.
The fact of word(s) searched in "Find in Search result" dialog excluded in the searching words list doesn't surprise me.

**Comment by sasumner** [comment]
@donho I agree it is a minor bug, because most users don't even know that you can "find in finder", but to me it seems reasonable that the shortcuts should move in the currently active Finder ...OR... when used and the main finder is not active, it should switch to the main finder before moving amongst the results.  Otherwise user is confused as to what is happening.

**Comment by sasumner** [comment]
@donho said:

>It's not really a bug to me.

Again, knowing that would have been more helpful at "issue time" than at "PR time".
At "issue time" the only effort expended is that which it takes to write up the issue.
By "PR time" someone has put coding/building/testing/debugging effort in, which can be significant.

**Comment by donho** [comment]
@sasumner 
I see your point. Let's keep it opened so we are aware of this issue.
OTOH, it won't be "fixed", since it's not a bug to me.
